modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/avg_metric.yml

```diff
@@ -175,3 +175,57 @@ setup:
             the_string_avg:
               avg:
                 field: string_field
+
+---
+"Counter field":
```

Comment:
So far, all tsdb related tests have been added under `rest-api-spec/test/tsdb` directory. I understand that this is the `average` aggregation, but I think it leaks decisions made for the tsdb object. 

I understand this is a matter of taste, but If was to look for counter related tests, I would be looking in tsdb related directories